IntelliTect.TestTools.TestFramework/IntelliTect.TestTools.TestFramework/IntelliTect.TestTools.TestFramework.csproj

```diff
@@ -2,20 +2,22 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Version>1.1.0-alpha.3</Version>
-    <PackageReleaseNotes>Add logging override
-Fix bug in TestBuilder causing it to not throw when expected</PackageReleaseNotes>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Authors>Mike Curn</Authors>
-    <Description>A framework for managing automated test steps, their dependencies, and the logging needed to help with debugging complex test scenarios</Description>
-    <PackageTags>test framework, unit test</PackageTags>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <WarningsAsErrors>NU1605, NU5105</WarningsAsErrors>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningLevel>4</WarningLevel>
+    <NoWarn>1701;1702;CA1303</NoWarn>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
```

Comment:
Same comment here, rather than duplicating this between debug and release build, just put the values inside of the default property group.

Reply:
Same as above. Moved.